docs/source/configuration/overview.mdx

```diff
@@ -1,16 +1,16 @@
 ---
-title: Configuring the Apollo Router
-subtitle: With environment variables, command-line options, and YAML file configuration
-description: Learn how to configure the Apollo Router with environment variables, command-line options and commands, and YAML configuration files.
+title: Configuring the GraphOS Router
```

Comment:
Same comment as above. This applies to Router Core as well

Reply:
Will edit the title and clarify in body it applies to both